sos/report/plugins/rhui.py

```diff
@@ -27,7 +27,7 @@ def setup(self):
             "/var/log/rhui-subscription-sync.log",
             "/var/cache/rhui/*",
             "/root/.rhui/*",
-            "/var/log/rhui/*",
+            "/var/log/rhui*/*",
```

Comment:
I would prefer explicitly listing individual directories. First, to ensure `sos` is collecting just what we need (and not e.g. "rhui-backup" directory a user created years ago). Second, to ensure the default limit 25MB of logfiles is applied to `rhui-installer` and to `rhui` directories *independently*.

Assume a case when `rhui` directory has >25MB logs. Then `rhui-installer` logs wont be collected (with the current PR), since when `sos` must truncate logs from one copy_spec, it truncates the oldest logs - what will be those from install time.

So I expect


    +            "/var/log/rhui-installer/*",

should be the patch.

Reply:
@pmoravec made the change to the original line and added the new line for the `/var/log/rhui-installer/*` directory.